multimedia/welle.io/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               qt5 1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+#PortGroup               compilers 1.0
+
+name                    welle.io
+categories              multimedia
+platforms               darwin
+maintainers             nomaintainer
+
+description             DAB/DAB+ Software Radio
+long_description        This is an open source DAB and DAB+ software defined radio (SDR)  \
+                        with support for rtl-sdr (RTL2832U) and airspy. It supports  \
+                        high DPI and touch displays and it runs even on cheap computers  \
+                        like Raspberry Pi 2/3 and 100€ China Windows 10 tablets.
+
+license                 GPL-2
+license_noconflict      openssl
+
+homepage                https://www.welle.io/
+
+github.setup            AlbrechtL welle.io 801e483ce5340cb816cff63743706c245d8c5fe6
+version                 2.0-beta3+git801e483
```

Comment:
Please move this to below the `PortGroup               cmake 1.1` line. This will allow you the remove the now redundant `name` line.